src/sdk/sql_cluster_router.cc

```diff
@@ -811,6 +811,62 @@ bool SQLClusterRouter::DropTable(const std::string& db, const std::string& table
         }
     }
 
+    // delete pre-aggr meta info if need
+    if (tableInfo.base_table_tid() > 0) {
+        std::string meta_db = openmldb::nameserver::INTERNAL_DB;
+        std::string meta_table = openmldb::nameserver::PRE_AGG_META_NAME;
+        std::string select_aggr_info =
+            absl::StrCat("select base_db,base_table,aggr_func,aggr_col,partition_cols,order_by_col,filter_col from ",
+                         meta_db, ".", meta_table, " where aggr_table = '", tableInfo.name(), "';");
```

Comment:
Is it possible that two different pre-aggregation tables have the same `table_name`? It seems there may be same deploy name under different dbs.

Reply:
it is a potential bug, will fix in this pr. `table_name` should be unique, otherwise, `create pre-aggr table` will fail. Temporarily add `base_db` to `table name`. ( it seems too long for a unique `table name`.